packages/web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/web-components",
-  "version": "15.0.0",
+  "version": "15.1.0",
```

Comment:
If there are no web-component changes, I don't think this should change.

Reply:
Looking at [this](https://github.com/department-of-veterans-affairs/component-library/compare/v47.0.0...main), it seems like there's one web-components commit that needs to go out?

Reply:
@micahchiang  oh good catch! For some reasons I thought it was just CSS Library.